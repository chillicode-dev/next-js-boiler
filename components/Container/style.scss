.Container {
  margin: 0 auto;
  padding: 0 15px;
  max-width: $containerMaxWidth * 1px;
  min-width: $containerMinWidth * 1px;
}
