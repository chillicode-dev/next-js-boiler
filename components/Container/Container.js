// Vendor
import React from 'react';
import PropTypes from 'prop-types';
import cn from 'classnames';
// Internals
import style from './style.scss';

const Container = ({className, children}) => {
  return <div className={cn(style.Container, className)}>{children}</div>;
};

Container.defaultProps = {
  className: '',
};

Container.propTypes = {
  className: PropTypes.string,
  children: PropTypes.node.isRequired,
};

export default Container;
