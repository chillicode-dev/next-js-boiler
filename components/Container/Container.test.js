import React from 'react';
import renderer from 'react-test-renderer';
import Container from './Container';

it('Renders Container correctly', () => {
  const tree = renderer.create(<Container>Text</Container>).toJSON();
  expect(tree).toMatchSnapshot();
});
