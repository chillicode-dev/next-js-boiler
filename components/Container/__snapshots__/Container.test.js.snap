// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders Container correctly 1`] = `
<div
  className="Container"
>
  Text
</div>
`;
